soccer/RobotConfig.cpp

```diff
@@ -33,14 +33,18 @@ RobotConfig::RobotConfig(Configuration* config, QString prefix)
       chipper(config, QString("%1/chipper").arg(prefix)),
       pivotVelMultiplier(new ConfigDouble(
           config, QString("%1/pivot/velMultiplier").arg(prefix))),
+      velMultiplier(new ConfigDouble(
+          config, QString("%1/translation/velMultiplier").arg(prefix), 1.0)),
+      angleVelMultiplier(new ConfigDouble(
+          config, QString("%1/rotation/velMultiplier").arg(prefix), 1.0)),
       accelerationMultiplier(new ConfigDouble(
           config, QString("%1/translation/accelMultiplier").arg(prefix))),
       minEffectiveVelocity(new ConfigDouble(
           config, QString("%1/translation/minEffectiveVelocity").arg(prefix),
           0.3)),
       minEffectiveAngularSpeed(new ConfigDouble(
           config, QString("%1/rotation/minEffectiveAngularSpeed").arg(prefix),
-          0.7)) {}
+          0.0)) {}
```

Comment:
No, I think 0.0 is a safer default. 0.7 was causeing alot of issues for a bit on the real robots.